src/get-started/java-binding.md

```diff
@@ -327,6 +327,7 @@ changes made to `Fork` to the database state:
 
 !!! warning
     Java integration tests that use the shared native library currently work on MacOS only.
+    The Java Binding App which runs Java services, however, works fine on Linux.
```

Comment:
I'd put the conjunction at the beginning:
However, the Java Binding App...
Also, does the Java Binding App work fine on both Linux and MacOS or only on Linux? If its the first case, I suggest the following wording:
However, the Java Binding App which runs Java services, works fine on both MacOS and Linux.